### PR TITLE
fix new bug for issue #1372

### PR DESCRIPTION
**Pull request by alpha-baby**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/network/listener.go [hunk 1]
```diff
@@ -220,15 +233,28 @@ func (l *listener) SetListenerTag(tag uint64) {
 }
 
 func (l *listener) ListenerFile() (*os.File, error) {
+	err := errors.New("not support this network " + l.network)
```
Comment (nejisama) [inline pkg/network/listener.go:236]: do not create an error every time. use bindport check instead?

Comment (nejisama) [reply pkg/network/listener.go:236]:
```
if  ! l.bindport {
    return nil, syscall.EINVAL
}
```

**Review by nejisama — APPROVED** [review]

File: pkg/network/listener.go [hunk 2]
```diff
@@ -256,6 +265,10 @@ func (l *listener) UseOriginalDst() bool {
 }
 
 func (l *listener) Close(lctx context.Context) error {
+	if !l.bindToPort {
```
Comment (nejisama) [inline pkg/network/listener.go:268]: l.state = ListenerStopped should be setted too
